scipy/stats/tests/test_multivariate.py

```diff
@@ -1024,6 +1026,68 @@ def test_wishart_invwishart_2D_rvs(self):
         assert_allclose(frozen_iw_rvs, manual_iw_rvs)
 
 
+class TestSpecialOrthoGroup(TestCase):
+    def test_reproducibility(self):
+        np.random.seed(514)
+        x = special_ortho_group.rvs(3)
+        expected = np.array([[ 0.99394515, -0.04527879,  0.10011432],
+                             [-0.04821555,  0.63900322,  0.76769144],
+                             [-0.09873351, -0.76787024,  0.63295101]])
+        assert_array_almost_equal(x, expected)
+
+    def test_frozen_matrix(self):
+        dim = 7
+        frozen = special_ortho_group(dim)
+
+        rvs1 = frozen.rvs(random_state=1234)
+        rvs2 = special_ortho_group.rvs(dim, random_state=1234)
+
+        assert_equal(rvs1, rvs2)
+
+    def test_det_and_ortho(self):
+        xs = [ special_ortho_group.rvs(dim)
+               for dim in range(2,12)
+               for i in range(3)
+             ]
+        # Test that determinants are always +1
+        dets = [ np.linalg.det(x) for x in xs ]
+        assert_allclose(dets, [1.]*30)
```

Comment:
`assert_allclose` has a rather measly default tolerance. I'd try to tighten it up.